### PR TITLE
FIT test for RackHD Redfish UpdateService API

### PR DESCRIPTION
**Pull request by dalebremner**

### CODE REVIEW

File: test/tests/redfish10/test_redfish10_api_updateservice.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+'''
+Copyright 2017 Dell Inc. or its subsidiaries.  All Rights Reserved.
+'''
+
```
Comment (hohene) [inline test/tests/redfish10/test_redfish10_api_updateservice.py]: Can you add a quick description to what this test does and verifies please?

File: test/tests/redfish10/test_redfish10_api_updateservice.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        fit_common.cancel_active_workflows(nodeid)
+        on_payload = {"ImageURI": "/dummy.exe", "Targets": [nodeid]}
+        api_data = fit_common.rackhdapi('/redfish/v1/UpdateService/Actions/UpdateService.SimpleUpdate',
+                                        action='post', payload=on_payload)
+        self.assertEqual(api_data['status'], 201,
+                         'Incorrect HTTP return code, expected 201, got:' + str(api_data['status']))
+        task_data = fit_common.rackhdapi(api_data['json']['@odata.id'])
+        self.assertEqual(task_data['status'], 200, "No task ID found ")
+        self.assertIn(task_data['json']['TaskState'], ["Running", "Pending", "Completed", "Exception"],
+                      "Bad task state for node:" + nodeid)
```
Comment (hohene) [inline test/tests/redfish10/test_redfish10_api_updateservice.py]: Can you print out the bad task state received in the error message?  Helps in debugging.
"Bad task state {} for node {}".format(task_data[....}, nodeid)

**Comment by dalebremner** [comment]
jenkins: test this please

**Comment by dalebremner** [comment]
@hohene all set...

**Comment by dalebremner** [comment]
jenkins: test this please

**Comment by dalebremner** [comment]
jenkins: test this please

**Pull request merged**